db/fixtures/aws_database_types.yml

```diff
@@ -1360,12 +1360,12 @@ db.r6i.2xlarge.tpc2.mem8x:
   :network_performance: :25_gbps
   :vcpu: 8
 db.r6i.32xlarge:
-  :deprecated: false
+  :deprecated: true
   :ebs_optimized: false
   :family: Memory optimized
   :memory: 1099511627776.0
   :name: db.r6i.32xlarge
-  :network_performance: :50_gbps
```

Comment:
Follow-up, I notice we handle `\d+ Gigabit` but not `\d+ Gbps`

Reply:
Good point, I'll look into this as a follow-up